lib/vmpooler/api/v1.rb

```diff
@@ -45,38 +45,75 @@ def alias_deref(hash)
       newhash
     end
 
-    def checkout_vm(template, result)
-      vm = backend.spop('vmpooler__ready__' + template)
+    def fetch_single_vm(template)
+      backend.spop('vmpooler__ready__' + template)
+    end
+
+    def return_single_vm(template, vm)
```

Comment:
This confused me at first - could we rename it to something like re_ready_single_vm or something along those lines?